truffle-config.js

```diff
@@ -16,7 +16,7 @@ if (!privateKey && !mnemonic) {
 
 // Solc
 let solcUseDocker = process.env.SOLC_USE_DOCKER === 'true' || false
-let solcVersion = '<0.5.5'
+let solcVersion = '<0.5.7'
```

Comment:
@anxolin 

Do you remember whether there was a particular reason to limit us to versions smaller than: 0.5.5?

Reply:
I think there might have been an issue compiling with 0.5.5 Maybe we don't need any restriction anymore?

Reply:
Ok, so since Adress requires `^0.5.5` we should also say `let solcVersion = '^0.5.5'`, right?

Reply:
Ther error message of this check here is super retarded, it was very hard to find.

I prefer the error message from the compiler, which will just state that it is not good enough for any version before 0.5.5